app/i18n/en/index.ts

```diff
@@ -735,6 +735,11 @@ const en: BaseTranslation = {
     learn: "I don't mean to badger you, but there's lot more to learn, dig in...",
     learnToEarn: "Learn to Earn",
   },
+  AuthIntroScreen: {
+    upgradeAccountRequirements: "Upgrading your account only requires a phone number",
+    signInCreateAccountRequirements: "Signing in or creating an account requires a phone number",
```

Comment:
should we say something like "come back and use the trial account if you want to explore the app to get started"?